spec/ui/actions.rst

```diff
@@ -0,0 +1,259 @@
+2. :term:`End User` selects bundle from "Bundle" selector in  :ref:`ui_form_dialogs_create_cluster`
+3. :term:`End User` selects version from "Version" selector in  :ref:`ui_form_dialogs_create_cluster`
+4. :term:`End User` fills "Cluster name" field in :ref:`ui_form_dialogs_create_cluster`
+5. :term:`End User` fills "Description" field in :ref:`ui_form_dialogs_create_cluster`
+6. :term:`End User` clicks "Create" button in :ref:`ui_form_dialogs_create_cluster`
+
+Post-Conditions
+---------------
+
+* A Cluster has been created.
+* :ref:`ui_form_dialogs_create_cluster` has beeen closed.
+
+Cluster service list view
+=========================
+
+Use case in which the user can view the list of added clusters and information on them.
+
+Actors
+------
+
+* :term:`End User`
+
+User Value
+----------
+
+This use case will allow the :term:`End User` view the list of services included to cluster.
+
+Pre-Conditions
+--------------
+
+:term:`End User` has ADCM and completed the following use cases:
+
+* :ref:`ui_usecases_login`
+* :ref:`ui_usecases_cluster_list_view`
+
+Post-Conditions
+---------------
+
+* :term:`End User` was able to view the list of cluster services and information on them.
+* Dialog has been closed
+
+“Used” Use Cases
+----------------
+
+Cluster host list view
+======================
+
+Use case in which the user can view the list of hosts added clusters.
+
+Actors
+------
+
+* :term:`End User`
+
+User Value
+----------
+
+This the only way to find out list of host included in cluster over UI.
+
+Pre-Conditions
+--------------
+
+:term:`End User` has ADCM and completed the following use cases:
+
+* :ref:`ui_usecases_login`
+* :ref:`ui_usecases_cluster_list_view`
+
+Post-Conditions
+---------------
+
+* :term:`End User` was able to view the list of cluster hosts.
+
+“Used” Use Cases
+----------------
+
+
+Flow of Events
+--------------
+
+1. :term:`End User` selects the required cluster by clicking on it in the list of clusters.
+2. :term:`End User` goes to the "Hosts" section in the Left Menu.
+3. :term:`End User` sees a list of cluster hosts.
+
+User Interface
+--------------
+
+List of host described in :ref:`ui_templates_cluster_host_list_view`.
+
+
+*************
+Host Provider
+*************
+
+
+.. _ui_usecases_create_hostprovider:
+
+Host Provider Create
+====================
+
+The way user create a Host Provider in UI.
+
+Actors
+------
+
+* :term:`End User`
+
+User Value
+----------
+
+That is the only way to create Host Provider.
+
+Pre-Conditions
+--------------
+
+:term:`End User` has ADCM and completed the following use cases:
+
+* :ref:`ui_usecases_login`
+* :ref:`ui_usecases_hostprovider_list_view`
+
+
+Flow of Events
+--------------
+
+1. :term:`End User` clicks "Create provider" button in :ref:`ui_form_dialogs_create_hostprovider`
+2. :term:`End User` selects bundle from "Bundle" selector in  :ref:`ui_form_dialogs_create_hostprovider`
```

Comment:
Samer here about bundle upload from this menu

Reply:
Don't what to do this in this scenario.